e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -370,6 +370,9 @@
               "IOT_HUB_CONNECTION_STRING": {
                 "value": "<IoTHubConnectionString>"
               },
+              "logUploadEnabled": {
```

Comment:
Need to add optimize performance flag here and replace value by script.

Reply:
No I don't. Please see the comment above.